osmoutils/binary_search_test.go

```diff
@@ -41,10 +41,11 @@ func TestBinarySearch(t *testing.T) {
 }
 
 func TestBinarySearchNonlinear(t *testing.T) {
-	// straight line function that returns input. Simplest to binary search on,
+	// x^2 function, a test example for a monotonic increasing function
 	// binary search directly reveals one bit of the answer in each iteration with this function.
-	lineF := func(a sdk.Int) (sdk.Int, error) {
-		return a, nil
+	expF := func(a sdk.Int) (sdk.Int, error) {
+		output := a.MulRaw(2)
+		return output, nil
```

Comment:
I'm confused, this replaced a line, with another line, not x^2 or an exponential function?